schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/message/MailMessage.java

```diff
@@ -14,7 +13,7 @@
  * @author krico
  * @since 26/11/14.
  */
-@Model(kind = "Msg", schemaVersionName = Constants.SCHEMA_VERSION_NAME, schemaVersion = 0)
+@Model
```

Comment:
I think I checked and we had some mails long ago but only a few and none in some months.